packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -86,7 +86,7 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<IDropdown,
    * Keys of the selected items. If you provide this, you must maintain selection
    * state by observing onChange events and passing a new value in when changed.
    */

```

Comment:
This should also say ` Note that passing in `null` will cause selection to be reset.`

Reply:
fixed